contrib/docker/docker-compose.yml

```diff
@@ -29,10 +34,13 @@ services:
       - traefik.port=8448
 
   db:
-    image: postgres:latest
+    image: postgres:10-alpine
```

Comment:
I'd suggest the same as you already did above: prepend `docker.io/`
I'd also do it in the Dockerfile, for the alpine base, whichever that'll be (alpine or python-alpine).